src/PythonProjectBootstrapper/package/hooks/post_gen_project.py

```diff
@@ -101,25 +101,12 @@ def SavePrompts() -> None:
         """,
     )
 
-{% if cookiecutter.sign_binaries %}
-    prompts["Create a Minisign Key"] = textwrap.dedent(
-        f"""\
-        In this step, we will create a Minisign (https://jedisct1.github.io/minisign/) key that is used to sign the binaries generated by GitHub Actions.
-
-        Note that these steps rely on docker (docker.com), so please make sure that docker is installed and working properly on your machine.
-        Additional information is available at https://docs.docker.com/engine/install/.
-
-        1. Run 'docker run -i --rm -v "{Path.cwd()}:/host" jedisct1/minisign -G -p /host/key.pub -s /host/key.pri -W'
-        2. Ensure that the file '{Path.cwd() / "key.pub"}' exists.
-        3. Ensure that the file '{Path.cwd() / "key.pri"}' exists.
-        """,
-    )
-
+{% if cookiecutter.minisign_public_key != 'none' %}

```

Comment:
Nit: Might be good to strip and make the key lowercase in case they enter "None" or "none "

Reply:
Good catch. I will add this to a separate commit after the other changes go in.